[zcash_client_sqlite] 'cargo check --no-default-features --features transparent-inputs --tests' fails

```
    Checking zcash_client_sqlite v0.17.2 (/media/ssd/home/daira/research/librustzcash/zcash_client_sqlite)
error[E0432]: unresolved imports `maybe_rayon::iter::IndexedParallelIterator`, `maybe_rayon::slice::ParallelSliceMut`
   --> zcash_client_sqlite/src/wallet/init/migrations/receiving_key_scopes.rs:282:16
    |
282 |         iter::{IndexedParallelIterator, ParallelIterator},
    |                ^^^^^^^^^^^^^^^^^^^^^^^ no `IndexedParallelIterator` in `iter`
283 |         slice::ParallelSliceMut,
    |         ^^^^^^^^^^^^^^^^^^^^^^^ no `ParallelSliceMut` in `slice`
    |
    = help: consider importing this trait instead:
            crate::ParallelSliceMut
help: a similar name exists in the module
    |
282 |         iter::{IntoParallelIterator, ParallelIterator},
    |                ~~~~~~~~~~~~~~~~~~~~
help: a similar name exists in the module
    |
283 |         slice::ParallelSlice,
    |                ~~~~~~~~~~~~~

warning: unused import: `ParallelIterator`
   --> zcash_client_sqlite/src/wallet/init/migrations/receiving_key_scopes.rs:282:41
    |
282 |         iter::{IndexedParallelIterator, ParallelIterator},
    |                                         ^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(unused_imports)]` on by default

error[E0599]: no method named `par_chunks_mut` found for struct `Vec<Option<(sapling_crypto::Node, Retention<BlockHeight>)>>` in the current scope
   --> zcash_client_sqlite/src/wallet/init/migrations/receiving_key_scopes.rs:744:26
    |
743 |                       let subtrees = sapling_commitments
    |  ____________________________________-
744 | |                         .par_chunks_mut(CHUNK_SIZE)
    | |_________________________-^^^^^^^^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
help: the following traits which provide `par_chunks_mut` are implemented but not in scope; perhaps you want to import one of them
    |
278 +     use crate::ParallelSliceMut;
    |
278 +     use rayon::slice::ParallelSliceMut;
    |
help: there is a method `par_rchunks_mut` with a similar name
    |
744 |                         .par_rchunks_mut(CHUNK_SIZE)
    |                          ~~~~~~~~~~~~~~~

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
warning: `zcash_client_sqlite` (lib test) generated 1 warning
error: could not compile `zcash_client_sqlite` (lib test) due to 2 previous errors; 1 warning emitted
```

--- a/zcash_client_sqlite/src/wallet/init/migrations/receiving_key_scopes.rs
+++ b/zcash_client_sqlite/src/wallet/init/migrations/receiving_key_scopes.rs
@@ -277,11 +277,11 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
 mod tests {
     use std::convert::Infallible;
 
+    use crate::ParallelSliceMut;
+    #[cfg(feature = "multicore")]
+    use maybe_rayon::iter::{IndexedParallelIterator, ParallelIterator};
+
     use incrementalmerkletree::Position;
-    use maybe_rayon::{
-        iter::{IndexedParallelIterator, ParallelIterator},
-        slice::ParallelSliceMut,
-    };
     use rand_core::OsRng;
     use rusqlite::{named_params, params, Connection, OptionalExtension};
     use tempfile::NamedTempFile;

